index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <!-- Nav Bar menu on mobile -->
+  <header>
+    <nav class="mobile-nav-bar">
+      <a href="">My Logo</a>
+      <img src="Assets/Unionhamb.png" alt="mobile menu">
+    </nav>
+  </header>
```

Comment:
Figma
![image](https://user-images.githubusercontent.com/78251927/234980790-70cee880-e1f8-4c36-b382-b09a83bd9b42.png)

Your Design
![image](https://user-images.githubusercontent.com/78251927/234980901-d697f85b-b79a-46d8-ac2a-d95270369074.png)

According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), it is important that our website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Kindly consider changing the style of the logo to match the given design. 🛠👍

[OPTIONAL] - Looking at the `Hamburger` menu in your design, it does not match the template 👀, it will be nice if you look into that and adjust it. 🛠👍